internal/github_client/types.go

```diff
@@ -22,3 +22,12 @@ type RateLimits struct {
 type GithubClient interface {
 	InitClient() *github.Client
 }
+
+type Account struct {
+	AuthType       string `json:"auth_type"`
```

Comment:
Maybe out of the scope of this PR, but I think we could consider making `AuthType` of a custom type so we can validate it's only one of `PAT` or `APP`, something like:

```go
type AuthType string

const (
	AuthTypePAT AuthType = "PAT"
	AuthTypeApp = "APP"
)

var ErrInvalidAuthType = errors.New("invalid auth type")

func (a AuthType) MarshalJSON() ([]byte, error) {
	return json.Marshal(string(a))
}

func (a *AuthType) UnmarshalJSON(b []byte) error {
	var s string
	if err := json.Unmarshal(b, &s); err != nil {
		return err
	}

	if s == "PAT"|| s== "APP"{
	*a = AuthType(s)
		return nil
	}

	return ErrInvalidAuthType
}
```

WDYT?

Reply:
I realised the app didn't fail on boot when receiving a bad auth type, as I would've expected. Instead, it let you boot up, but then crashed when trying to collect the metrics, which seemed... _inconvenient_

I've added this, and also made it fail in that case, for both single and multiple accounts flows